### ISSUE TITLE
[Carousel]: Accessibility issue - Section with role="listbox" missing a label

### ISSUE THREAD

**Issue by MaksymPakhomov** [SAP/ui5-webcomponents#8472]
### Bug Description

Siteimprove Accessibility Checker encountered the following error: 4.1.2 Name, Role, Value
The form field missing a label. Every form element should have a descriptive text label.
Add a label, making sure it's properly associated with the form field.

![image](https://github.com/SAP/ui5-webcomponents/assets/149803486/c82a117a-6bf7-4b63-a6b3-d3a101663d46)

Thanks a lot in advance!

### Affected Component

Carousel

### Priority

Medium

### UI5 Web Components Version

1.23.1

### Browser

Chrome

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rd 
can you follow up on the issue, it's reproducible.

**Comment by ilhan007** [comment]
will be available within days with 1.24
